.evergreen/config.yml

```diff
@@ -408,26 +408,6 @@ axes:
         variables:
           ASYNC: "false"
           FRAMEWORK: "netcoreapp2.1"
-      - id: "dotnet-async-netcoreapp1.1"
```

Comment:
it was decided to reduce the number of used variants to reduce the atlas loading